src/components/ComponentNames.ts

```diff
@@ -0,0 +1,3 @@
+export enum ComponentName {
```

Comment:
Should the file name be lower casre componentNames.ts ?

Reply:
I generally keep the files the same as the export which in this case is CapitalCase given it's an enum. I am open to whatever we want

Reply:
I think you should stick to airbnb's recommnded  https://github.com/airbnb/javascript/tree/master/react#naming
Lower case unless is a component name .tsx
also https://stackoverflow.com/a/49655115/340827